src/ui/public/agg_types/metrics/derivative.js

```diff
@@ -0,0 +1,142 @@
+import AggTypesMetricsMetricAggTypeProvider from 'ui/agg_types/metrics/metric_agg_type';
+import orderAggTemplate from 'ui/agg_types/controls/sub_agg.html';
+import _ from 'lodash';
+import $ from 'jquery';
+import VisAggConfigProvider from 'ui/vis/agg_config';
+import VisSchemasProvider from 'ui/vis/schemas';
+
+export default function AggTypeMetricDerivativeProvider(Private) {
```

Comment:
every other agg type is using default export ... what do you think would be better? keeping this similar to other aggs ? or going with named export ? (which wont be used as this is only exporting a single thing)